### PR TITLE
Admin vm shared service

### LINKED ISSUE #2611: Make an admin-vm shared service
## Description

As a TRE Administrator,
I want to install an admin-vm as a shared service
So that it will be created only when required

## Acceptance criteria

- [ ] Extract admin-vm from core infrastructure 
- [ ] Remove tools that aren't required and won't be possible to use (shared services are installed after the fw and most traffic will be blocked)
- [ ] Confirm admin tasks are still possible

### PR DESCRIPTION
**Pull request by tamirkamara**
# Resolves #2611 

## What is being addressed

The admin vm deployed in the core infra isn't used often and sometimes it also fail the provisioning.

## How is this addressed

- Remove it from the core deployment and recreated as an optional shared service that admins could install on demand.
- No tools will be installed on this new service

### CODE REVIEW

File: templates/shared_services/admin-vm/template_schema.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/microsoft/AzureTRE/templates/shared_services/gitea/template_schema.json",
```
Comment (anatbal) [inline templates/shared_services/admin-vm/template_schema.json:3]: The template shouldn't be this path
```suggestion
  "$id": "https://github.com/microsoft/AzureTRE/templates/shared_services/admin-vm/template_schema.json",
```

File: templates/shared_services/admin-vm/terraform/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (anatbal) [inline templates/shared_services/admin-vm/terraform/deploy.sh:1]: Do we need deploy.sh, destory.sh?

**Review by anatbal — APPROVED** [review]
Two minor comments

File: templates/shared_services/admin-vm/terraform/deploy.sh (hunk 2 above)
Comment (tamirkamara) [reply templates/shared_services/admin-vm/terraform/deploy.sh:1]: yes, for local testing.

**Pull request merged**